core/src/main/java/com/datastax/oss/driver/api/core/tracker/RequestTracker.java

```diff
@@ -46,28 +47,46 @@ default void onSuccess(
       @NonNull DriverExecutionProfile executionProfile,
       @NonNull Node node) {}
 
+  /**
+   * @deprecated This method only exists for backward compatibility. Override {@link
+   *     #onSuccess(Request, long, DriverExecutionProfile, Node, ExecutionInfo, String)} instead.
+   */
+  @Deprecated
+  default void onSuccess(
+      @NonNull Request request,
+      long latencyNanos,
+      @NonNull DriverExecutionProfile executionProfile,
+      @NonNull Node node,
+      @NonNull String requestLogPrefix) {
+    // If client doesn't override onSuccess with requestLogPrefix delegate call to the old method
+    onSuccess(request, latencyNanos, executionProfile, node);
+  }
+
   /**
    * Invoked each time a request succeeds.
    *
    * @param latencyNanos the overall execution time (from the {@link Session#execute(Request,
    *     GenericType) session.execute} call until the result is made available to the client).
    * @param executionProfile the execution profile of this request.
    * @param node the node that returned the successful response.
+   * @param executionInfo the execution info containing the results of this request
    * @param requestLogPrefix the dedicated log prefix for this request
    */
   default void onSuccess(
       @NonNull Request request,
       long latencyNanos,
       @NonNull DriverExecutionProfile executionProfile,
       @NonNull Node node,
+      @NonNull ExecutionInfo executionInfo,
       @NonNull String requestLogPrefix) {
-    // If client doesn't override onSuccess with requestLogPrefix delegate call to the old method
-    onSuccess(request, latencyNanos, executionProfile, node);
+    // If client doesn't override onSuccess with executionInfo delegate call to the old method
```

Comment:
@lukasz-antoniak and I thought we don't need the parameter `request`, `executionProfile`, and `node` here, because they are all contained in `executionInfo` object. So we can change it to:
```java
  default void onSuccess(
      long latencyNanos,
      @NonNull ExecutionInfo executionInfo,
    onSuccess(executionInfo.getRequest(), latencyNanos, executionInfo.getExecutionProfile(), executionInfo.getCoordinator(), requestLogPrefix);
  }
```
If we leave it like it is, the user may be confused if the ones in the parameters and the ones in `exeuctionInfo` Object are not consistent. 
@absurdfarce @adutra How do you think?

Reply:
Good point.  As long as we retain and deprecate the old method signatures for backwards-compatability, I'm good.

Reply:
agreed, I like the concise method signature as well!

Seeing `onSuccess(long latencyNanos, @NonNull ExecutionInfo executionInfo, String requestLogPrefix)` makes me wonder why we couldn't just put latency on `ExecutionInfo`?   That seems like it could be useful for those consuming `ExecutionInfo` on their result set.

I think it's fine as proposed though :+1:

Reply:
We may consider putting `requestLogPrefix` into `ExecutionInfo`, too.